.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
-  "semi": false,
-  "trailingComma": "es5",
-  "singleQuote": false,
-  "printWidth": 120,
-  "endOfLine": "auto",
-  "arrowParens": "always",
-  "plugins": ["prettier-plugin-tailwindcss"]
+    "printWidth": 120,
+    "tabWidth": 4,
```

Comment:
I personally find tabWidth 4 to become very annoying with JSX because of the heavy indentation.

I also don't think that it's worth re-formatting every part of the codebase for this.